### PR TITLE
OAS-5911 | add rebranding changes

### PR DESCRIPTION
**Pull request by adoi**

### CODE REVIEW

**Review by osilva — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ oasisctl -h
 
 ## Authentication
 
-Oasisctl uses an authentication token to authenticate with the ArangoDB Oasis platform.
+Oasisctl uses an authentication token to authenticate with the ArangoGraph Insights Platform (formerly called Oasis) platform.
```
Comment (Robert-Stam) [inline README.md:40]: last platform, after the ) can be dropped imo.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
 # Oasisctl
 
-![ArangoDB Oasis](https://cloud.arangodb.com/assets/logos/arangodb-oasis-logo-whitebg-right.png)
+<img src="https://cloud.arangodb.com/assets/logos/arangograph-logo-auth.svg" alt="ArangoGraph Insights Platform" width="400"/>
 
-Commandline utility for accessing ArangoDB Oasis.
+Commandline utility for accessing ArangoGraph Insights Platform (formerly called Oasis).
 
-This utility is used to access ArangoDB Oasis resources (such as projects, deployments, certificates) without the needs for a graphical dashboard.
+This utility is used to access ArangoGraph Insights Platform (formerly called Oasis) resources (such as projects, deployments, certificates) without the needs for a graphical dashboard.
```
Comment (Robert-Stam) [inline README.md:7]: bit overkill I see: ,please remove (formerly called Oasis)

File: README.md [hunk 3]
```diff
@@ -37,7 +37,7 @@ oasisctl -h
 
 ## Authentication
 
-Oasisctl uses an authentication token to authenticate with the ArangoDB Oasis platform.
+Oasisctl uses an authentication token to authenticate with the ArangoGraph Insights Platform (formerly called Oasis).
```
Comment (Robert-Stam) [inline README.md:40]: bit overkill I see: ,please remove (formerly called Oasis)

**Review by Robert-Stam — COMMENTED** [review]
please remove on 2 places

**Comment by adoi** [comment]
#138 @Robert-Stam added it here

**Pull request merged**